src/pages/programs/index.tsx

```diff
@@ -19,6 +20,12 @@ import OtherPrograms from '../../images/hero-additional-programs.jpg';
 const ProgramsPage: NextPage = () => (
   <DefaultLayout>
 
+    <SEO
+      title="Courses &amp; Professional Training Programs"
```

Comment:
We shouldn't escape HTML entities inside javascript strings. This will literally render as "Courses &amp; Professional Training Programs"

Reply:
@davewelsh This was fixed. It should be ready to be merged now.